journal/modernized/20-chapter-17.adoc

```diff
@@ -583,6 +583,7 @@ hurricane in the neighbourhood of the Cape of Good Hope.
 This circumstance spread a gloom over us;
 having witnessed such dreadful weather about three weeks before,
 when the mercury was at 29'50,
+//The 29'50 was circled in the book, I'm not sure why
```

Comment:
ok, so after a few minutes googling (Because i have no idea about how a barometer works), I'm pretty confident that we should slightly change this to `29.50`.  We could also add a quotation mark (to indicate _inches_), but it is not necessary, and the comma afterwards makes it awkward. Probably let's change it in the _original_ edition too, and then you can remove this comment as well.